build.sbt

```diff
@@ -308,8 +308,8 @@ lazy val cli = project
 
             def zipEntries(zipStream: ZipInputStream): Iterator[(ZipEntry, Array[Byte])] =
               new Iterator[(ZipEntry, Array[Byte])] {
-                var nextEntry = Option.empty[ZipEntry]
-                def update() =
+                private var nextEntry = Option.empty[ZipEntry]
+                private def update() =
```

Comment:
It helped me to understand the code and so I hope others as well later, so I tend to keep it. But this is a very weak tendency.